packages/tools/nodeRenderGraphEditor/src/main.scss

```diff
@@ -0,0 +1,370 @@
+#node-render-graph-editor-graph-root {
+    display: grid;
+    grid-template-rows: calc(100% - 120px) 120px;
+    height: 100%;
+    width: 100%;
+    background: #464646;

```

Comment:
```suggestion
    background: #460046;

```

Reply:
This didn't change anything, so I also updated line 53 from #5f655b to #5f005b, though I'm not sure the color is ok (a bit too dark in my opinion)...

Reply:
Agreed we need a darker one. Feel free to pick one. This is your baby!

Reply:
![image](https://github.com/user-attachments/assets/532d7fa0-9966-4648-b3f3-24c9529e714a)